src/ha_addon_sunsynk_multi/a_sensor.py

```diff
@@ -189,7 +189,7 @@ def on_change(val: float | int | str | bool) -> None:
             ent["icon"] = "mdi:clock"
             self.entity = SelectEntity(
                 **ent,
-                options=sensor.available_values(15, ist.get_state),
+                options=sensor.available_values(OPT.number_entity_mode, ist.get_state),
```

Comment:
Should this not be `OPT.prog_time_interval`?

Reply:
yep, you are correct